libs/nx-terraform/src/common/getTfResourceName.ts

```diff
@@ -6,24 +6,28 @@ export async function getTfResourceName(
     projectRoot: string,
 ) {
     try {
-        const outputJson = await execa(
+        const outputHcl = await execa(
             'terragrunt',
             [
                 'state',
```

Comment:
Maybe this `state` command isn't correct.

https://www.terraform.io/cli/commands/show#json-output

Reply:
That doesn't allow you to filter by the resource so you end up with a multil 1000 line json output that you have to filter through

Reply:
and it's multi level